docfx.json

```diff
@@ -508,11 +508,9 @@
         "docs/core/deploying/**.md": ".NET Core",
         "docs/core/docker/**.md": ".NET Core",
         "docs/core/migration/**.md": ".NET Core",
-        "docs/core/porting/**.md": ".NET Core",
```

Comment:
The articles under `porting` folder don't need suffix at all. All titles already have ".NET Core".
This is the list of the titles:

- Port from .NET Framework to .NET Core
- Port libraries to .NET Core
- .NET Framework technologies unavailable on .NET Core
- Organize projects for .NET Framework and .NET Core
- Analyze dependencies to port code to .NET Core
- Tools for porting to .NET Core
- Use the Windows Compatibility Pack to port code to .NET Core
- Breaking changes for WinForms - .NET Framework to .NET Core
- Port a Windows Forms app to .NET Core

Reply:
The problem with removing it is that we might add a new article to the porting folder that doesn't have .NET Core in its title. So I think a better solution here is to leave it in docfx.json and instead modify the title and/or titleSuffix metadata in the individual files.

Reply:
That will require adding 9 empty titleSuffix for all the articles under porting. I see that majority of the articles will have ".NET Core" in the title  (Currently, all of them do). So, I don't see having a docfx titleSuffix that's overwritten by **all** (currently), (or **most** in future), articles. If a new article needed the suffix, the suffix can be added in the article itself. Personally, I'll prefer using docfx global setting if the major files will use it, and minor files will overwrite it. Not the opposite.

But anyways, if you still want to use the docfx setting and overwrite it in all files, I'll do that.